WorkloadManagementSystem/Utilities/QueueUtilities.py

```diff
@@ -0,0 +1,225 @@
+"""Utilities to help Computing Element Queues manipulation
+"""
+
```

Comment:
I think any new files should start with:
```suggestion
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
```
These imports will help avoid new Python 3 incompatibilities being introduced and, unlike `unicode_literals`, are contained to a single file.

Reply:
If we put it into the DIRAC.__init__ - will it be by definition applied in all the modules ? Or it should be put in each module ?

Reply:
It's applied on a per file basis which is what makes it safe to use here and for any other new files in DIRAC. Applying the division import everywhere would probably cause (and fix) bugs. Hopefully #4240 will help smooth the transition.